helios-tools/src/main/java/com/spotify/helios/cli/command/HostListCommand.java

```diff
@@ -107,6 +112,11 @@ int run(final Namespace options, final HeliosClient client, final PrintStream ou
         .from(client.listHosts().get())
         .filter(containsPattern(pattern))
         .toList();
+    HostStatus.Status statusFilter = null;
+    try {
+      statusFilter = HostStatus.Status.valueOf(options.getString(statusArg.getDest()));
+    } catch (IllegalArgumentException | NullPointerException ignored) {
```

Comment:
erroring out here would probably be nice, and avoid confusion when you have typos.

E.g. "list --status up" or "list --status DWN" will list all hosts but not actually filter anything out